scripts/components/graphs/line-graph.js

```diff
@@ -103,7 +103,27 @@ class LineGraph {
 				const offset = 'position: ' + (isX ? `${dist}px 0px 0px;` : `0px ${dist}px 0px;`);
 
 				// Linear interpolate to determine marker text, backwards for Y. Then round to precision and cast back to Number.
-				const markerValue = +(isX ? lineMin + j : lineMax + lineMin - j).toFixed(precision);
+				let markerValue = +(isX ? lineMin + j : lineMax + lineMin - j).toFixed(precision);
+				if (!isX && axis.name === $.Localize('#Run_Stat_Name_Time')) {
+					const extreme = Math.max(Math.abs(axis.min), Math.abs(axis.max));
+					const time = Math.abs(markerValue);
+
+					let sign = Math.sign(markerValue) >= 0 ? '+' : '-';
+					if (markerValue === 0) sign = '';
```

Comment:
I actually really think a nested ternary is more readable here. `const sign = markerValue > 0 ? '+' : markerValue < 0 ? '-' : '';`